docs/installation/docker.rst

```diff
@@ -144,6 +146,8 @@ Keep in mind that ``POSTGRES_PASSWORD`` and ``DB_PASSWORD`` have to be *identica
 
 Also keep in mind that ``APP_KEY`` must be *exactly* 32 characters long.
 
+Additionally, Postgres uses the username as the database name.
+
```

Comment:
Please fix the English grammar here

Reply:
I've copied and pasted from a GitHub comment you've made in the past.
What do you propose?

Reply:
For example, the sentence above, grammatically, should be:
"Also -> **,** <- keep in mind that ``APP_KEY`` must be *exactly* 32 characters long."

Reply:
It's fine, sorry, I read it wrong :grin: